spec/controllers/snippets_controller_spec.rb

```diff
@@ -12,93 +12,93 @@
   it { should use_before_action(:check_show_permissions) }
   it { should use_before_action(:redirect) }
 
-  describe "GET #index" do
-    it "should redirect users to login_path if not log in" do
+  describe "#index" do
+    it "not logged in" do
```

Comment:
Yea i was thinking of the context. But it gets bulkier fast. So i decided to just not include it. Ill change it in the next pr